services/graph/apps/worker/src/worker.module.ts

```diff
@@ -34,18 +26,34 @@ import { QueueModule } from '#lib/queues/queue.module';
       // set this to `true` if you want to emit the removeListener event
       removeListener: false,
       // the maximum amount of listeners that can be assigned to an event
-      maxListeners: 10,
+      maxListeners: 20,
```

Comment:
Ah, here's the `maxListeners` update for graph. Is this tuned per service? I see it's either 10 or 20.

Reply:
I probably tuned it based on whether I saw an error/warning or not.

Reply:
(I believe the default is 10)